app/components/events/index.js

```diff
@@ -0,0 +1,38 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action  } from '@ember/object';
+
+export default class EventsComponents extends Component{
+
+	/** variable for data*/
+
+	//meeting variable for all meeting related details like meeting links
+	meeting = {
+		link: 'https://www.airmeet.com/e/57d5b2b0-c7a7-11ea-a4ea-fd3d36d2c7f6'
+	};
+
+	//event variable for all details for the upcoming event
+	event = {
+		title: "How to build for Web using EmberJS",
```

Comment:
Extremely NIT: Different quotes (" and ') in different places

Reply:
Still confused with this. So replaced all single quotes with double quotes.